auth2-gss.c

```diff
@@ -265,10 +265,6 @@ input_gssapi_exchange_complete(int type, u_int32_t plen, struct ssh *ssh)
 
 	authenticated = PRIVSEP(ssh_gssapi_userok(authctxt->user));
 
-	if ((!use_privsep || mm_is_monitor()) &&
```

Comment:
can you remove this?

Reply:
If you're asking is it safe to remove, then I think the answer is yes because this is only ever called in packet context.

Reply:
If you're instead asking if I can delete mm_is_monitor() entirely, then I'm not sure because of session.c:do_cleanup()

Reply:
yes, I was talking about mm_is_monitor.